XenAdmin/Wizards/GenericPages/SelectVMStorageWithMultipleVirtualDisksPage.cs

```diff
@@ -284,9 +291,23 @@ public override void PopulatePage()
 
 	    private void PopulateSrComboBox(List<ToStringWrapper<SR>> commonSRs)
 	    {
-	        List<EnableableSrComboboxItem> listToAdd = 
-	            commonSRs.Select(toStringWrapper => new EnableableSrComboboxItem(toStringWrapper, m_totalSpaceRequired)).ToList();
-
+			List<EnableableSrComboboxItem> listToAdd = new List<EnableableSrComboboxItem>();
+			foreach (var toStringWrapper in commonSRs)

```

Comment:
This calculation obsoletes the calculation for `m_totalSpaceRequired`. We should remove this field as it is assigned but never used. When removing, please note that there is a reference to it in a comment as well.